1config-ui/src/com/brunobonacci/oneconfig/ui/view.cljs

```diff
@@ -47,7 +47,7 @@
   (for [item items]
     (let [{:keys [key env version change-num content-type]} item]
       ^{:key (string/join "-" [key env version change-num content-type])}
-      [:tr {:class "center aligned"}
+      [:tr {:class (str "center aligned " (string/join "-" [key env version change-num content-type]))}
```

Comment:
i don't understand the need of this change here

Reply:
I use it for checkboxes.

Reply:
yeah, you are correct! It is one of my previous attempts, each checkbox had the unique selector.
After that, I improved the solution and use `[parent (-> e .-target .-parentElement .-parentElement)]`, so I guess it can be removed